frontend/src/pages/projects/CreateProject.jsx

```diff
@@ -3,15 +3,10 @@ import App from "../../App";
 import { useSelector, useDispatch } from "react-redux";
 import { useNavigate } from "react-router-dom";
 import ProjectTypeSelect from "./ProjectTypeSelect";
-import {
-    setProjectId,
-    setProjectShortTitle,
-    setProjectTitle,
-    setProjectDescription,
-    setSelectedProjectType,
-} from "./createProjectSlice";
+import { setProjectId, setProjectShortTitle, setProjectTitle, setProjectDescription } from "./createProjectSlice";
 import { useAddResearchProjectsMutation } from "../../api/opsAPI";
 import Alert from "../../components/UI/Alert/Alert";
+import { Modal } from "../../components/UI/Modal/Modal";
 
 export const CreateProject = () => {
     const [currentIndex, setCurrentIndex] = React.useState(0);
```

Comment:
Linting is yelling about these, are they used?

Reply:
Don't think so - removed it.